aes/decryptor.go

```diff
@@ -19,6 +19,11 @@ type encryptedValue struct {
 
 const nonceSize int = 32
 
+type stashData struct {
+	iv        []byte
+	plaintext interface{}
+}
+
```

Comment:
Callers don't need to have access to this struct. The stash is basically an arbitrary structure each cipher can use for bookkeeping, and the sops package (or any other callers) shouldn't touch it at all. They should just save it somewhere when they decrypt, and pass it back when they want to encrypt the same value.